Sources/ApodiniAuthorization/AuthorizationRequirements/RequirementResult.swift

```diff
@@ -0,0 +1,41 @@
+//
+// Created by Andreas Bauer on 09.07.21.
+//
+
+/// A ``RequirementResult`` represents the result of an evaluation of an ``AuthenticationRequirement``.
+public enum RequirementResult {
+    /// Represents a positive ``RequirementResult``.
+    /// Captures the ``Cause`` leading to the result. The ``Cause`` is only captured for debug purposes.
+    case fulfilled(cause: @autoclosure () -> Cause = .unspecified)
+    /// Represents a neutral ``RequirementResult``.
+    /// Captures the ``Cause`` leading to the result. The ``Cause`` is only captured for debug purposes.
+    case undecided(cause: @autoclosure () -> Cause = .unspecified)
```

Comment:
`Authenticator` interprets this case as successful. I don't think that is obvious. Please add an according note to the docs.

Reply:
I'll add some docs 👍